_posts/en/newsletters/2023-12-06-newsletter.md

```diff
@@ -0,0 +1,177 @@
+---
+title: 'Bitcoin Optech Newsletter #280'
+permalink: /en/newsletters/2023/12/06/
+name: 2023-12-06-newsletter
+slug: 2023-12-06-newsletter
+type: newsletter
+layout: newsletter
+lang: en
+---
+This week's newsletter FIXME
+
+## News
+
+- **Cluster mempool discussion:** Bitcoin Core developers working on
+  [cluster mempool][topic cluster mempool] started a [working
+  group][wg-cluster-mempool] (WG) on the Delving Bitcoin forum. Cluster
+  mempool is a proposal to make it much easier to compare the
+  profitability of different valid orderings of transactions.  A valid
+  ordering of Bitcoin transactions requires parent transactions get
+  confirmed before their children, either by putting the parent in an
+  earlier block than any of its children, or earlier in the same block.
+  _Clusters_ of one or more related transactions are designed to be easy
+  to include into feerate-sorted _chunks_ that are each independent of
+  the other chunks. For example, we might split a mempool into 100
```

Comment:
I think there might be misunderstanding here. A _cluster_ refers to the connected component of a transaction, i.e. all ancestors and descendants of a transaction all of the ancestors and descendants of all those, plus all of the ancestors and descendants of all that, etc. until there is nothing to add. _Chunks_ refer to sections of a linearization of a cluster. 

Linearizations must be topologically valid. They are chunked by first making every transaction its own chunk, then combining any two chunks A and B, if A is the left neighbor of B and B has a higher feerate than A. Ideally, linearizations are sorted to produce the highest possible feerate diagram across the cluster, but non-optimal linearizations may occur.

We are currently expecting that clusters might be limited to 100 transactions, much like the descendant and ancestor limit in the current mempool design, and since chunks are subcomponents of a cluster’s linearization, they would usually not be whole blocks.